### PR TITLE
Create library.json

### PR DESCRIPTION
**Pull request by vazquezjm**
library.json is a manifest file of development library for PlatformIO. PlatformIO Library Crawler uses library.json manifest to extract source code from developer’s location and keeps a cleaned library in own Library Registry. 
https://docs.platformio.org/en/latest/librarymanager/config.html#library-json

### CODE REVIEW

File: library.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "SevenSegmentTM1637",
+  "version": "1.0.0",
```
Comment (pfeerick) [inline library.json]: 1.0.4, unless you _want_ an older version ;)

File: library.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "keywords": "TM1637, TM1636, Arduino, LED, AVR, Atmega, esp8266, IC",
+  "description": "Library for using a 4 digit seven segment display with TM1636 or TM1637 driver IC",
+  "repository":
+  {
+    "type": "git",
+    "url": "https://github.com/bremme/arduino-tm1637.git"
+  },
+  "authors":
```
Comment (pfeerick) [inline library.json]: The authors block can also be omitted as this is on github, and the library manager will populate it itself... also allowing it to self-update if anything changes

File: library.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        "email": "bramharmsen@gmail.com"
+    },
+    {
+        "name": "Andrew Smith",
+        "email": "me@andrew-smith.com",
+        "maintainer": true    	
+    }
+  ],
+  "frameworks": "arduino",
+  "platforms": "avr"
```
Comment (pfeerick) [inline library.json]: Think this should be `atmelavr`, but you mentioned esp8266 in the tags, so should probably be at least `atmelavr, espressif8266`... or you can just do the `*` wildcard and see what platforms it doesn't work on :-P :laughing:

File: library.json (hunk 1 above)
Comment (vazquezjm) [reply library.json]: Right! My bad. This is my first PR

**Comment by per1234** [comment]
What is the purpose of adding a library.json file when this library already has a library.properties file, which is supported by PlatformIO as an alternative to library.json?:
https://docs.platformio.org/en/latest/userguide/lib/cmd_register.html#description

Having two metadata files only doubles the maintenance burden and makes it likely that two two will end up out of sync with no benefit that I can see. Since the Arduino IDE does not support library.json, it seems the obvious choice is to just use a single library.properties file.

**Comment by pfeerick** [comment]
@per1234 Thanks for that... gonna have to do a PR on the docs to fix that... as it clearly states that the library manifest file is the only requirement to register a library... but only mentions the json file... not that others are supported.

> The only one requirement is library’s manifest file - library.json.

File: library.json (hunk 2 above)
Comment (vazquezjm) [reply library.json]: They're now gone

File: library.json (hunk 3 above)
Comment (vazquezjm) [reply library.json]: As you suggested, I used * ;)

**Comment by vazquezjm** [comment]
Hi @bremme! Is there any chance that you could merge this PR? This file will let PlatformIO user add the library from Visual Studio Code among other things. Thanks!

**Comment by per1234** [comment]
@vazquezjm did you miss what I said in my previous reply about library.json being unnecessary due to this library already having a library.properties file? Or is there something I don't understand about PlatformIO that makes it still necessary to have a library.json file?

**Pull request closed without merging**